File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int n= nums.length;
        int count =1;
      //  int[] = temp;

        for(int i=1;i<n;i++){
            if(nums[i]!=nums[i-1]){
                nums[count]=nums[i];
                count++;
            }
        }return count;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        c=0
        n=len(nums)
        for i in range (1,n):
            if nums[i]!=nums[c]:
                c+=1
                nums[c]=nums[i]
        return c+1

        
